src/to-do.js

```diff
@@ -0,0 +1,127 @@
+import { check } from './validate.js';
+
+const InnerTodo = document.querySelector('.todo-body');
+let isEdit = false;
+let editId = null;
+
+export default class display {
+  static getTodo = () => {
+    let Todo;
+    if (localStorage.getItem('todo') == null) {
+      Todo = [];
+    } else {
+      Todo = JSON.parse(localStorage.getItem('todo'));
+    }
+    return Todo;
+  };
```

Comment:
I think if you rewrite this as:
```
static getTodo = () => {
   if (localStorage.getItem('todo') {
   return JSON.parse(localStorage.getItem('todo'));
   }
   return [];
}
```

Reply:
Ok thanks for the feedback, I will implement that right away👌😃😃